core/src/test/java/org/elasticsearch/action/search/InnerHitsCollapseIT.java

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
For a test like this we prefer the rest tests:
https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/test/search/110_field_collapsing.yml
It will start nodes that you can query like a client would do in the real world.

Reply:
Thanks for your quick response. I pushed c9b8ee488ddabf807932d2441ac8ff9536f58a6d